n2survey/lime/survey.py

```diff
@@ -143,6 +147,34 @@ class LimeSurvey:
                 "A5": "severe depression",
             },
         },
+        "noincome_duration": {
+            "label": "How long working on PhD without income?",
```

Comment:
Please write complete sentences as labels. You can even copy paste the original question labels because here the answers are "essentially" changing dtype nothing more.